Framework/PythonInterface/mantid/api/src/Exports/ExperimentInfo.cpp

```diff
@@ -80,7 +86,7 @@ void export_ExperimentInfo() {
            "file will encapsulate this date (e.g '1900-01-31 23:59:00')\n"
            "\nreturns : list of absolute paths for each valid file\n")
       .staticmethod("getResourceFilenames")
-      .def("getInstrumentFilename", &ExperimentInfo::getInstrumentFilename,
+      .def("getInstrumentFilename", &FileFinderHelpers::getInstrumentFilename,
```

Comment:
Should we consider adding a deprecation warning to this pointing to the new class? We can then remove it in the future?